bblab.dockerfile

```diff
@@ -53,45 +66,48 @@ RUN R -e "install.packages(\"https://cran.r-project.org/src/contrib/Archive/rvch
 RUN rm -rf /tmp/downloaded_packages/ /tmp/*.rds \
     rm -rf /var/lib/apt/lists/*
 
-# set the timezone for Vancouver, so that datetime.now() returns our
-# local time, not UTC.
-RUN apt-get -y install tzdata
-ENV TZ=America/Vancouver
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
-
 # we define these users here so that we can launch an image as a local user
 # see the Makefile in the top directory...
-RUN useradd -rm -s /bin/tcsh -u 1000 dockuser00 &&\
-    useradd -rm -s /bin/tcsh -u 1001 dockuser01 &&\
-    useradd -rm -s /bin/tcsh -u 1002 dockuser02 &&\
-    useradd -rm -s /bin/tcsh -u 1003 dockuser03 &&\
-    useradd -rm -s /bin/tcsh -u 1004 dockuser04
+RUN useradd -rm -s /bin/tcsh -u 1000 dockuser00 ; \
+    useradd -rm -s /bin/tcsh -u 1001 dockuser01 ; \
+    useradd -rm -s /bin/tcsh -u 1002 dockuser02 ; \
+    useradd -rm -s /bin/tcsh -u 1003 dockuser03 ; \
+    useradd -rm -s /bin/tcsh -u 1004 dockuser04 ; true
```

Comment:
Are you sure it's OK to ignore errors from creating these users?

Reply:
These lines seem to only matter for development, but I never needed them, and they sometimes failed docker build. I would even remove them.

Reply:
Removing them sounds better than leaving them in, if they're unreliable and unneeded.

Reply:
Ok, I removed them